fluent-plugin-enhance-k8s-metadata/lib/sumologic/kubernetes/connector.rb

```diff
@@ -0,0 +1,60 @@
+module SumoLogic
+  module Kubernetes
+    # module for connecting to Kubernetes cluster
+    module Connector
```

Comment:
is it possible to re-use it in the events plugin?

Reply:
It's possible, but need to split the code into a mixin so different plugins can refer to. 
Or you can just copy the code into your plugin first with same module names and later we can merge and change the dependencies